lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -212,6 +207,19 @@ div.course-wrapper {
           }
         }
       }
+
+      .vert > .xblock-student_view {
+        @extend .clearfix;
+        border-bottom: 1px solid #ddd;
+        margin-bottom: ($baseline*0.75);
+        padding: 0 0 15px;
+      }
+
+      .vert > .xmodule_HiddenModule {
+          display: none;
+          border-bottom: 0px;
+          margin-bottom: 0px;
+      }
```

Comment:
@marcotuts We need to be able to hide blocks. If a block type is to be hidden, the platform returns an empty div for its html. Currently the html of a block goes into `xblock-student_view`. However `.vert`, it parent, has some margin and border associated with it. Since we would not want them if the div is empty I have moved the sass from `.vert` to `.xblock-student_view` and neutralized `.vert > .xmodule_HiddenModule` (which would be the class of the empty div). Does this all make sense?